active/0030-default-security-improvements.md

```diff
@@ -0,0 +1,102 @@
+# Default security improvements
+
+## Changelog
+
+* 2025-06-30: @id Initial draft
+
+## Abstract
+
+This EIP proposes to improve the default security of EMQX.
+
+## Motivation
+
+The default security settings of EMQX can be improved to follow best practices and enhance the security of out-of-the-box installations.
+
+## Design
+
+1. Bind the Dashboard’s listener to 127.0.0.1 by default.
+
+2. Bind MQTT and Websocket listeners to 127.0.0.1 by default.
```

Comment:
Binding to loopback is not going to help a lot in docker deployments (most common pattern for non-production, experimental deployment) due to port forwarding.

Maybe focus on authn/authz defaults.